content/github/managing-your-work-on-github/managing-labels.md

```diff
@@ -49,6 +49,8 @@ Label | Description
 
 Default labels are included in every new repository when the repository is created, but you can edit or delete the labels later.
 
+Issues with the `good first issue` label are used to populate the repository's contribute page.
```

Comment:
```suggestion
Issues with the `good first issue` label are used to populate the repository's `contribute` page. For an example of a `contribute` page, see [github/docs/contribute](https://github.com/github/docs/contribute)
```

Reply:
Just added a link to an example contribute page, as mentioned in the content design plan!